src/set_blob_metadata_request_base.cpp

```diff
@@ -0,0 +1,41 @@
+#include "set_blob_metadata_request_base.h"
+
+#include "constants.h"
+#include "utility.h"
+
+namespace azure {  namespace storage_lite {
+
+    void set_blob_metadata_request_base::build_request(const storage_account &a, http_base &h) const
+    {
+        const auto &r = *this;
+
+        h.set_absolute_timeout(5L);
```

Comment:
>5L [](start = 31, length = 2)

I know it aligns with other non-IO-heavy operations. But can you still make it a constant instead of a magic number for the newly added code? Even `DEFAULT_SHORT_TIMEOUT` is better then `5L`.

Reply:
I think we'd better either change all these timeout stuff or no change at all. It'll be confusing if some requests come with a timeout while some do not.

Reply:
I would suggest we change all, it should be a simple 'replace all' operation.

Reply:
No, I don't think `DEFAULT_SHORT_TIMEOUT` is a good idea, actually I don't think the unexplained constant number is a good idea. One might ask why it isn't 6 seconds or 4?

I think it's a better idea if client side doesn't set timeout if users don't explicitly asked to. But this would be a breaking change.